### PR TITLE
Create a ConfirmationModal

### PR DESCRIPTION
**Pull request by harshnstack**
Create a ConfirmationModal

image

image

### CODE REVIEW

File: src/stories/components/Modal/Modal.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [open, setOpen] = React.useState(false);
+
+  const handleClose: any = () => {
+    setOpen(false);
+  };
+  return (
+    <>
+      <div className="Modal">
+        <div
```
Comment (jaynstack) [inline src/stories/components/Modal/Modal.tsx:37]: you should use box component of mui for this pleas dont use div on mui library

Comment (harshnstack) [reply src/stories/components/Modal/Modal.tsx:37]: okay sure, I will change it

File: src/stories/components/Modal/ConfirmationModal.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+export default {
+  title: "ConfirmationModal",
+  component: ConfirmationModal,
+} as unknown as ComponentMeta<typeof ConfirmationModal>;
+
+const Template: ComponentStory<typeof ConfirmationModal> = (args) => (
+  <ConfirmationModal {...args} />
+);
+
+export const Primary = Template.bind({});
```
Comment (jaynstack) [inline src/stories/components/Modal/ConfirmationModal.stories.tsx:15]: please give a proper name to a component not like primary and secondary

Comment (harshnstack) [reply src/stories/components/Modal/ConfirmationModal.stories.tsx]: I changed it please check

Comment (harshnstack) [reply src/stories/components/Modal/ConfirmationModal.stories.tsx]: Please also follow code format in component name like logincomponent , searchelement and toastmessage

**Review by jaynstack — APPROVED** [review]

**Pull request merged**